project/plugins.sbt

```diff
@@ -1,7 +1,11 @@
 addSbtPlugin("de.heikoseeberger"                 % "sbt-header"       % "1.5.1")
 addSbtPlugin("org.scalariform"                   % "sbt-scalariform"  % "1.5.1")
-addSbtPlugin("com.jsuereth"                      % "sbt-pgp"          % "1.0.0")
-addSbtPlugin("com.dwijnand"                      % "sbt-dynver"       % "1.0.0")
+addSbtPlugin("com.dwijnand"                      % "sbt-dynver"       % "1.0.0+15-3655d014")
 addSbtPlugin("com.lightbend.paradox"             % "sbt-paradox"      % "0.2.5")
 addSbtPlugin("com.eed3si9n"                      % "sbt-unidoc"       % "0.3.3")
-addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "0.2.4")
+
+// Reenable after https://github.com/ThoughtWorksInc/sbt-api-mappings/issues/5 is fixed and released
+//addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "0.2.4")
+
+addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0-8-g6d0c3f8")
+resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/2m/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
```

Comment:
So he can inject his hacked artifacts! 😜 
Probably an omission? If not then should be documented